specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionsCreateOrUpdate.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "connectionName": "testManagedApi",
+    "api-version": "2016-06-01",
+    "apiConnection": {
+      "properties": {
+        "displayName": "testManagedApi",
+        "statuses": [
+          {
+            "status": "Connected"
+          }
+        ],
+        "customParameterValues": {},
+        "nonSecretParameterValues": {
```

Comment:
i believe this is internal id, not to be disclosed

Reply:
Removed